vllm/v1/worker/tpu_model_runner.py

```diff
@@ -239,6 +241,12 @@ def __init__(
         self.structured_decode_arange = torch.arange(
             0, 32, device="cpu", pin_memory=self.pin_memory)
 
```

Comment:
It seems multi-lora is disabled on tpu "The V1 TPU backend doesn't support LoRA serving". Do you know the reason why we do that?

Reply:
Ah yeah I disabled it here until we have #16989 in because that has the best runtime/compile time performance